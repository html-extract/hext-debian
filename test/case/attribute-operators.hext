<div class="swo" class*="wor" class$="word" class^="swo" same=="same" class:class/>
