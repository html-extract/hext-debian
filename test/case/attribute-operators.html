<div id="anchor">
  <div same="same" class="swo sword"></div>
  <div same="same" class="swo ssword"></div>
  <div class="ord"></div>
  <div class="or"></div>
  <div class="nope"></div>
</div>
