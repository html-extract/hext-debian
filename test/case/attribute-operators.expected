{"class":"swo ssword"}
{"class":"swo sword"}
